Makefile

```diff
@@ -198,13 +198,19 @@ define SHOW_ARCH
   $(shell printf "\\t%s%s\\\n" "$(1)" $(if $(filter $(ARCH),$(1))," (default)",""))
 endef
 
-all: runtime
+all: vc runtime
```

Comment:
A couple of points:

- `vc` and `install-vc` need to be added to `show-usage`.

- `make clean` doesn't undo `make vc`.